src/functionaltests/resources/features/authentication.feature

```diff
@@ -37,16 +37,18 @@ Feature: Test Authentication
   @RESTWithTokenId
   Scenario: Call an REST API with session credentials
     Given LDAP is activated
+    When a GET request is prepared for REST API "/auth/logout"  
```

Comment:
Should this repeat?

Reply:
This is a logout, the one below is a login